tests/test_neural_modules_import.py

```diff
@@ -0,0 +1,114 @@
+# ! /usr/bin/python
```

Comment:
Can you rename this file too? again configuration import

Reply:
ok, creating a separate PR